### DISCUSSION TITLE
One license for Firefox on both Windows and Mac?

### DISCUSSION THREAD

**Discussion by gregconquest** [aclap-dev/video-downloadhelper#886] [Help and Q&A]
### Your question

I prefer to use VDH on Mac, but it seems that before the newest version, anyway, that many sites work fine with the Windows version but not the Mac version of VDH. I hope this improves, but I'm willing to continue running the Windows version (via Bootcamp) to download videos.

But if I buy a license, can I use it on both Mac and Window versions of VDH?

### Addon & CoApp version and details

version: 9.0.2.3
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:127.0) Gecko/20100101 Firefox/127.0
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "/Users/gregconquest/media_editing/me_ecques",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 3,
    "successfull_dl": 1
}

**Comment by GrampaWildWilly** [comment]
#744

**Comment by paulrouget** [comment]
> But if I buy a license, can I use it on both Mac and Window versions of VDH?

Yes.
